### ISSUE TITLE
columnName issue in sails-postgresql\node_modules\waterline-sequel\sequel\lib\criteriaProcessor.js

### ISSUE THREAD

**Issue by hemangj** [balderdashy/sails-postgresql#150]
i am using columnName in model as shown in below exmaple:
```node
        student_city: {
            columnName: 'city',
            type: 'string',
            required: true
        }
```

Now sails-postgresql\node_modules\waterline-sequel\sequel\lib\criteriaProcessor.js:
function: CriteriaProcessor.prototype.process = function process(parent, value, combinator, caseSensitive) {...}
        It is uses columnName (city) while if condition self.currentSchema[parent]. So condition failed because columnName(city) is not found in self.currentSchema. 'student_city' is present. 
        So lower value is false and case sensitive query is fired. 
         If I will remove columnName and update database then lower value is true and case insensitive query is fired. 
         Can anyone look this issue?

**Comment by devinivy** [comment]
Are you referencing this block of code?
https://github.com/balderdashy/waterline-sequel/blob/master/sequel/lib/criteriaProcessor.js#L200-L206

**Comment by hemangj** [comment]
Thanks for your quick reply.

I am using sails-postgresql npm module, version is 0.10.14. It is internally uses waterline-sequel npm module and version is 0.1.1.

https://github.com/balderdashy/waterline-sequel/blob/0.1.1/sequel/lib/criteriaProcessor.js#L359-L364

**Comment by sailsbot** [comment]
Thanks for posting, @hemangj.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
